### PR TITLE
readme

### PR DESCRIPTION
**Pull request by RxnDbr**

### CODE REVIEW

**Comment by armgilles** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -16,47 +21,32 @@ npm install
...
 ````
 
+Then, you must copy create the missing file
 
+````
+cd 6element/client/src
+cp mapbox mapbox-creadentials.json.example mapbox-creadentials.json
+````
 
+Sign up to [mapbox](https://www.mapbox.com/), a free app using OpenStreetMap.
```
Comment (DavidBruant) [inline README.md]: No need to talk about OSM

File: README.md [hunk 2]
```diff
@@ -16,47 +21,32 @@ npm install
...
-
-### Learning algo
-
-In `predict` lies the learning algorithm. Once you get the data, enter the ipython prompt and use:
-
-```
-%run contextualize.py
-%run learn.py
-```
+In a new consol
```
Comment (DavidBruant) [inline README.md]: console

File: README.md [hunk 3]
```diff
@@ -16,47 +21,32 @@ npm install
...
+npm start
 
-The algorithm concerning the computer vision part are in `vision`.
-You can use the car detector or the bin detector individually, but for a complete analysis use:
+````
 
-```
-%run analyse_all.py
-```
+You can now open the URL return
```
Comment (DavidBruant) [inline README.md]: `returned URL`

**Pull request merged**